Sources/Testing/ExitTests/ExitTest.Condition.swift

```diff
@@ -41,49 +56,71 @@ extension ExitTest {
 
 // MARK: -
 
-@_spi(Experimental)
 #if SWT_NO_EXIT_TESTS
 @available(*, unavailable, message: "Exit tests are not available on this platform.")
 #endif
 extension ExitTest.Condition {
-  /// A condition that matches when a process terminates successfully with exit
-  /// code `EXIT_SUCCESS`.
+  /// A condition that matches when a process exits normally.
+  ///
+  /// This condition matches the exit code `EXIT_SUCCESS`.
+  ///
+  /// @Metadata {
+  ///   @Available(Swift, introduced: 6.2)
+  /// }
   public static var success: Self {
     Self(_kind: .success)
   }
 
-  /// A condition that matches when a process terminates abnormally with any
-  /// exit code other than `EXIT_SUCCESS` or with any signal.
+  /// A condition that matches when a process exits abnormally
+  ///
+  /// This condition matches any exit code other than `EXIT_SUCCESS` or any
+  /// signal that causes the process to exit.
+  ///
+  /// @Metadata {
+  ///   @Available(Swift, introduced: 6.2)
+  /// }
   public static var failure: Self {
     Self(_kind: .failure)
   }
 
-  public init(_ statusAtExit: StatusAtExit) {
-    self.init(_kind: .statusAtExit(statusAtExit))
+  /// @Metadata {
+  ///   @Available(Swift, introduced: 6.2)
+  /// }
+  public init(_ exitStatus: ExitStatus) {
```

Comment:
I notice this `init` for `ExitTest.Condition` doesn't have any DocC symbol documentation. Often for "trivial" initializers we omit this, but since it's a `public` API, it looks somewhat sparse in our rendered documentation, and it seems like we at least usually have docs for all _public_ initializers. I'll add some basic documentation for this.

Reply:
Go for it!